fcrepo-auth-webac/src/main/java/org/fcrepo/auth/webac/WebACFilter.java

```diff
@@ -325,7 +329,13 @@ private boolean isAuthorized(final Subject currentUser, final HttpServletRequest
                     return false;
                 }
             } else {
-                return currentUser.isPermitted(toRead);
+                if (currentUser.isPermitted(toRead)) {
+                    if (!isAuthorizedForContainedResources(httpRequest, currentUser, resource)) {
+                        log.debug("GET denied, not authorized for an embedded resource");
```

Comment:
Can you update this `debug` message to include the `resource`? ...and maybe `currentUser`?

Reply:
This message also applies to "OPTIONS" and "HEAD"... not just "GET" as the `debug` statement suggests.